python_modules/libraries/dagster-dg/dagster_dg/utils/__init__.py

```diff
@@ -575,10 +581,12 @@ def parse_json_option(context: click.Context, param: click.Option, value: str):
 # ########################
 
 TomlPath: TypeAlias = tuple[Union[str, int], ...]
-TomlDoc: TypeAlias = Union[tomlkit.TOMLDocument, dict[str, Any]]
+TomlDoc: TypeAlias = Union["tomlkit.TOMLDocument", dict[str, Any]]
 
 
 def load_toml_as_dict(path: Path) -> dict[str, Any]:
+    import tomlkit
```

Comment:
lot of these in this general utils file is it possible to move these to their own file and recurse up one and have less deferred imports or something? im not sure how worth it is since a lot of dg will be getting rewritten if were dropping all the cross process stuff